sdk/ml/azure-ai-ml/azure/ai/ml/entities/_inputs_outputs/utils.py

```diff
@@ -240,7 +240,16 @@ def _update_io_from_mldesigner(annotations: dict) -> dict:
     to IO entities.
     """
     from azure.ai.ml import Input, Output
+    from .enum_input import EnumInput
```

Comment:
No need to update this function, this is a compatible code block to enable old mldesigner package which does not have _Param at all

Reply:
It needs to keep this change. If user uses the primitive annotation type in dsl.pipeline, it needs to convert it to Input class here.